rules/keep-track-of-a-parking-lot-for-topics/rule.md

```diff
@@ -1,22 +1,23 @@
 ---
 type: rule
-archivedreason:
-title: Do you keep track of a "Parking Lot" for off topic items that are important, but need to be taken offline?
-guid: c9bda201-180b-4dc2-8fb7-9c770ab09844
+title: Do you keep track of a "Parking Lot" for off topic items that are
+  important, but need to be taken offline?
 uri: keep-track-of-a-parking-lot-for-topics
-created: 2021-07-20T06:35:55.0000000Z
-authors: 
-- title: Ulysses Maclaren
-  url: https://ssw.com.au/people/uly
+authors:
+  - title: Ulysses Maclaren
+    url: https://ssw.com.au/people/uly
 related:
-- methodology-daily-scrums
-
+  - methodology-daily-scrums
+created: 2021-07-20T06:35:55.000Z
+archivedreason: null
+guid: c9bda201-180b-4dc2-8fb7-9c770ab09844
 ---
-
 Make sure that you cover as much of the agenda as possible within the timebox, and keep other topics to a minimum.
 
-When you identify topics that do need to be hashed out, but are off topic and don’t need to be covered in this meeting, keep track of them in a “Parking Lot”.  
-E.g. During the meeting you can call out _“Off topic. I think that topic should be saved for the “Parking Lot”. Who is interested in joining that?”_
+When you identify topics that do need to be hashed out, but are off topic and don’t need to be covered in this meeting, keep track of them in a “Parking Lot”.
+
+E.g. During the meeting you can call out *“Off topic. I think that topic should be saved for the “Parking Lot”. Who is interested in joining that?”*
 
 <!--endintro-->
 
+![Figure: Video - See the Parking Lot in action: www.youtube.com/watch?v=-UUrLxNBK_g](parking-lot.jpg)
```

Comment:
Embed the YouTube video. See www.ssw.com.au/rules/rule (about halfway down the page)
```suggestion
`youtube: www.youtube.com/watch?v=-UUrLxNBK_g`
**Figure : See the parking lot in action**
```